spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -886,7 +827,7 @@ def slice_installer_options(host)
         allow( subject ).to receive( :on ).with( host, /puppet agent -t/, :acceptable_exit_codes => [0,2] )
       end
       #  We wait for puppetdb to restart 3 times; once before the first puppet run, and then during each puppet run
-      allow( subject ).to receive( :sleep_until_puppetdb_started ).with( hosts[0] ).exactly(3).times 
+      allow( subject ).to receive( :sleep_until_puppetdb_started ).with( hosts[0] ).exactly(3).times
```

Comment:
Kinda nit-picky but is there any reason why this change needs to be in this commit/PR? It seems unrelated and kinda messes with git history of this file.